.github/CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # See: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
 
 # Default owners or everything in the repo. Unless a later match takes precedence,
-*	@BFH-ktt1 @maehne @MarcinOrlowski
+*	@BFH-ktt1 @maehne @MarcinOrlowski @zdimension
```

Comment:
What the purpose for this change?

Reply:
@zdimension has made considerable contributions over the past year. I am therefore not opposed to this change and would leave the final decision to @BFH-ktt1.

Reply:
That's fine with me, but I am asking as this is solely for auto-assigning code reviewers of newly created PR. If  @zdimension is willing to review PRs then sure, go ahead as these can sometime stack up a small pile, otherwise I'd suggest not to.

Reply:
I am willing to, this is why I proposed this change to the CODEOWNERS file.

Reply:
Perfect, thanks for clarification.